roles/openshift_logging_kibana/templates/5.x/kibana.j2

```diff
@@ -81,7 +81,7 @@ spec:
             periodSeconds: 5
         -
           name: "kibana-proxy"
-          image: {{ proxy_image }}
+          image: "{{ openshift_logging_kibana_proxy_image_prefix }}logging-auth-proxy:{{ openshift_logging_kibana_proxy_image_version }}"
```

Comment:
does the auth proxy need a 5.x image build?

Reply:
not really.  It could be pegged to the same one used in 2.x